### ISSUE TITLE
BackWPup version mismatch

### ISSUE THREAD

**Issue by NedbaseBastiaan** [outlandishideas/wpackagist#478]
Hi there!

The BackWPup plugin recently had an issue in their latest release (4.0.0), which made them decide to completely remove that version from the Wordpress SVN and fall back to (3.10.0). However WPackagist already had the 4.0.0 version indexed, which meant that all our projects still pulled the version of the plugin that didn't exist anymore. Now the problem has got a bit 'worse', as the plugin recently released a new version of their plugin again under the 4.0.0 tag. The version of the plugin on WPackagist however is still the old 4.0.0 version. Is there perhaps something that you guys could do to 're-index' the 4.0.0 tag? We already asked the plugin maker to release a new version (4.0.1) to solve this, but they are not responding and it is really hurting our development proces in the meantime.

So in short, the release of the tags was as followed:
- plugin maker released 4.0.0 tag (which caused errors)
- plugin maker deleted 4.0.0 tag and went back to 3.10.0 (to fix errors)
- plugin maker released a fix under the same 4.0.0 tag (everything should be fixed now)

Perhaps you guys can't fix this at all, but I guess it wouldn't hurt to ask :) 
Thanks in advance!

**Comment by NoelLH** [comment]
Hey @NedbaseBastiaan, sorry for the late reply.

I can see 4.0.0 is still the latest tag, but wanted to check if you're still having problems with the Wpackagist version?

There is a "refresh" option that anyone can press on the [web results](https://wpackagist.org/search?q=BackWPup&type=any&search=) and it looks like somebody already did so last week, so I want to check first if that has resolved the problem. If not, I can investigate further.

**Comment by NoelLH** [comment]
I'll close this for now, but please reopen with the current state of play if you're still having problems.
